hivemq-edge/src/frontend/src/components/rjsf/Form/error-focus.utils.ts

```diff
@@ -0,0 +1,107 @@
+import { MutableRefObject } from 'react'
+import Form from '@rjsf/core'
+import { RJSFSchema, RJSFValidationError, UiSchema } from '@rjsf/utils'
+import { ChakraRJSFormContext, UITab, UITabIndexed } from '@/components/rjsf/Form/types.ts'
+
+export const _toPath = (path: string) => path?.match(/([^[.\]])+/g)
+
+// From RJSF
+/* istanbul ignore next -- @preserve */
+const focusOnError = (formElement: HTMLFormElement, error: RJSFValidationError) => {
+  // const { idPrefix = 'root', idSeparator = '_' } = props;
+  const idPrefix = 'root',
+    idSeparator = '_'
+
+  const { property } = error
+  if (!property) return
+
+  const path = _toPath(property)
+  if (!path) return
+  if (path[0] === '') {
```

Comment:
I see that you're checking whether there's a path. But what you're not checking yet is that `path` is an array and that its length is > 0

Reply:
the regex match takes care of that